tools/c7n_left/c7n_left/cli.py

```diff
@@ -0,0 +1,48 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+#
+import logging
+from pathlib import Path
+
+import click
+from c7n.config import Config
+
+from .output import get_reporter, report_outputs
+from .provider import CollectionRunner
+from .utils import load_policies
+
+
+@click.group()
+def cli():
+    """Shift Left Policy"""
+    logging.basicConfig(level=logging.DEBUG)
+
+
+@cli.command()
+@click.option("--format", default="terraform")
+@click.option("-p", "--policy-dir", type=click.Path())
+@click.option("-d", "--directory", type=click.Path())
+@click.option("-o", "--output", default="cli", type=click.Choice(report_outputs.keys()))
+@click.option("--output-file", type=click.File("w"), default="-")
```

Comment:
This feels a bit disorienting coming from typical custodian usage, but seems to make sense in the terraform context. Not sure if we would also expect any per-policy or per-resource output where providing an output dir might be more useful.

Reply:
yeah.. its a different interface/ux then standard custodian which is part of the desire on separate cli, I looked around at a few tools, and this style seems to be the most common (re single output file)